package.json

```diff
@@ -53,7 +53,7 @@
     "opn": "^5.2.0",
     "pify": "^3.0.0",
     "prettier": "^1.11.1",
-    "sinon": "^5.0.0",
+    "sinon": "^4.5.0",
```

Comment:
Downgrade? why?

Reply:
If you install 5.0, they say it's deprecated.  I checked `npm info sinon`, and it looks like they accidentally released it at one point, then quickly switched back the dist tag to 4.5. I could pull this out to another PR if you'd like :)

Reply:
Thanks for explaining. Sounds fine here.